### PR TITLE
Update master to warrior

### PR DESCRIPTION
**Pull request by htot**
This is mostly an update to Yocto Warrior + U-Boot 2019-10 + Linux 5.4.0.
It also restores some functionality from Factory:
- clloader
- arduino-toolchain (this lays some groundwork but still needs work)
- ota
- building debian images 

And then there are a few fixes and updates to the docs.

### CODE REVIEW

**Comment by htot** [comment]
@alext-mkrs do you  think you will have time to review this?

**Comment by alext-mkrs** [comment]
That would be a bit hard as I've dismantled my yocto build setup a while ago and haven't touched my Edisons even longer. Would just a "visual" review work? I could try to carve out some time, but that've been hard last [many] months too, TBH...

**Comment by htot** [comment]
@alext-mkrs visual inspection may do for now. Another user already tested and we've been ironing out remaining problems here: https://github.com/htot/meta-intel-edison/pull/4

Too bad you have so little time, Edison is really starting to work well. In the meantime I started a local branch based on Zeus and converting to btrfs to use the emmc more efficiently.

**Comment by alext-mkrs** [comment]
Yeah, it's unfortunately the matter of priorities in view of very limited free time... That and the fact that I currently have no good use for Edison is basically the reason, but as you can see, I try to be around :) On it now.

File: docs/_docs/Edison/Setting Up/1.2-Setting-up.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ product: Edison
 
         git checkout master
 
-You can also checkout dizzy-uptodate, dizzy-latest, dizzy-rt or morty, pyro64, pyro64-acpi, rocko32, rocko64-acpi, sumo32 and sumo64-acpi (all frozen) or thud
+You can also checkout dizzy-uptodate, dizzy-latest, dizzy-rt or morty, pyro64, pyro64-acpi, rocko32, rocko64-acpi, sumo32 and sumo64-acpi, thud (all frozen) or warrior.
 
-Master/thud is currently a continuation of sumo32 with acpi enabled by default and for thud with enabled x86_64 on top of that.
+Master/warrior is currently a continuation of sumo32 with acpi enabled by default and for  with enabled x86_64 on top of that.
```
Comment (alext-mkrs) [inline docs/_docs/Edison/Setting Up/1.2-Setting-up.md]: typo: missing word in the "and for  with enabled x86_64" piece (was thud previously)

File: docs/_docs/Edison/Setting Up/1.3-ACPI-or-not.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ product: Edison
 * TOC
 {:toc}
 ## ACPI or no-ACPI?
-{% include note.html content="In Thud ACPI is switched on by default and the `acpi` removed from `DISTRO_FEATURES`. For instructions on building a non-ACPI version checkout the sumo32 branch. A good reason would be if the ACPI doesn't cover your needs yet and you need to build a version without. This would be the case if you need to use the MRAA and UPM librairies" %}
+{% include note.html content="Since Thud ACPI is switched on by default and the `acpi` removed from `DISTRO_FEATURES`. For instructions on building a non-ACPI version checkout the sumo32 branch. A good reason would be if the ACPI doesn't cover your needs yet and you need to build a version without. This would be the case if you need to use the MRAA and UPM librairies" %}
```
Comment (alext-mkrs) [inline docs/_docs/Edison/Setting Up/1.3-ACPI-or-not.md]: typo (an old one): librairies

File: utils/debian_1_create.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ROOTDIR=$1
+if [ -z "$ROOTDIR" ]
+then
+echo "Select a distribution" ||
+exit 1
+fi
+
+cd out/linux64/build
+
+sudo debootstrap --arch amd64 --no-check-gpg $ROOTDIR $ROOTDIR http://http.debian.net/debian/
```
Comment (alext-mkrs) [inline utils/debian_1_create.sh]: I'd suggest using a HTTPS-base mirror instead, see e.g. a list [here](https://gist.github.com/eighthave/7285154#gistcomment-2820716)

File: docs/_docs/Edison/Hacking/5.2-Documentation.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+title: Creating a deb repository
```
[the remark this answers is not in the document]
Comment (htot) [reply docs/_docs/Edison/Hacking/5.2-Documentation.md]: Copy/paste

File: utils/debian_1_create.sh (hunk 3 above)
Comment (htot) [reply utils/debian_1_create.sh]: Ok I selected https://deb.debian.org/debian and leave it to users to find a faster mirror near them.

This server behaves strange if you use your browser and redirects to http mirror, but seems to work fine with debootstrap.

**Comment by htot** [comment]
I captured your comments in 2 patches, thanks for reviewing!

**Pull request merged**